templates/CSI/cephfs/pvc.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ pvc_name | default('cephfs-pvc') }}
```

Comment:
It can go in CSI/pvc.yaml and both rbd and fs can utilize it, the defaults can be set to None so that we force user to use fs or rbd.